ocaml/forkexecd/test/fe_test.ml

```diff
@@ -20,6 +20,13 @@ let min_fds = 7
 
 let max_fds = 1024 - 13 (* fe daemon has a bunch for its own use *)
 
+let fail x =
```

Comment:
Is it ok to shadow the first definition of fail?

Reply:
It's fine, they're defined one after the other so it's obvious which one is being called

Reply:
Style question: is it preferred to shadow or define an inner function with another name and use it?
This shadow way reduces the diff, the first `fail` is identical to the "old" `fail` (just moved in this case to get reused more times).

Reply:
It depends, in short functions I prefer nesting; in longer or deeply nested functions it makes the function harder to read so it's better to define functions further up.